assets/css/settings.scss

```diff
@@ -535,3 +535,103 @@
 		}
 	}
 }
+
+@font-face {
+	font-family: 'Reforma 1918-Blanca';
+	src: url( '../css/fonts/Reforma1918-Blanca.otf' );
+	font-weight: normal;
+	font-style: normal;
+}
+
+
+.woocommerce-banner {
+	display: flex;
+	flex-direction: column;
+	margin-top: 10px;
+}
+
+@media screen and (min-width: 1100px) {
+	.woocommerce-banner {
+		display: flex;
+		flex-direction: row;
+		margin-top: 20px;
+	}
+}
+
+#woothemes-sensei .woocommerce-banner {
+	&__body {
+		color: #FFFFFF;
+		font-size: 16px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 26px;
+		letter-spacing: 0em;
+		text-align: left;
+		margin-bottom: 20px;
+		max-width: 420px;
+	}
+
+	&__header {
+		color: #6fcfb2;
+		font-family: 'Reforma 1918-Blanca';
+		font-size: 32px;
+		font-style: normal;
+		font-weight: 400;
+		line-height: 45px;
+		letter-spacing: 0em;
+		text-align: left;
+		margin-bottom: 16px;
+	}
+
+	&__background {
+		background-color: #26212e;
+		display: flex;
+		flex-direction: column;
+		max-height: 269px;
+		box-sizing: border-box;
+	}
+	@media screen and (max-width: 1500px) {
```

Comment:
I don't think this needs to be inside a media query. Removing it would mean 24px will be applied up until a screen size of 1500px, after which it will be 48px.